From 5b859cca580ee9c050486024ebd8cfdb34049008 Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <frantisek@sumsal.cz>
Date: Mon, 23 Jan 2023 19:13:49 +0100
Subject: [PATCH] test: bump the container spawn timeout to 60s

As 30s might be not enough on busy systems (and we already bumped the
reboot timeout from 30s to 60s for this reason).

(cherry picked from commit d932022ddfe021b1c49ffaf4d7dfe4093656f0c5)

Related: #2138081
---
 test/test-shutdown.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/test-shutdown.py b/test/test-shutdown.py
index e181f976be..13e18ecbaa 100755
--- a/test/test-shutdown.py
+++ b/test/test-shutdown.py
@@ -17,7 +17,7 @@ def run(args):
     logger.info("spawning test")
     console = pexpect.spawn(args.command, args.arg, env={
             "TERM": "linux",
-        }, encoding='utf-8', timeout=30)
+        }, encoding='utf-8', timeout=60)
 
     if args.verbose:
         console.logfile = sys.stdout
